tests/integration/config_test.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+include './tests/unit/utils.sh'
+include './tests/integration/utils.sh'
+
+function oneTimeSetUp()
+{
+  local distro
+
+  # bring up the containers
+  setup_container_environment
+
+  # copy config files to the containers
+  for distro in "${DISTROS[@]}"; do
+    container_copy "kw-${distro}" "${SAMPLES_DIR}/config" /tmp/.kw
+  done
+}
+
+# The current workaround is this: we are using specially crafted config files
+# that overwrite all global options in the same order displayed by kw.
+function local_config_test_helper()
+{
+  local config="$1"
+  local config_file
+  local container
+  local distro
+  local expected_output
+
+  # make sure config is defined
+  assertTrue 'config is not defined' "[[ -n '${config}' ]]"
```

Comment:
I think you can remove the `assertTrue` from here and move this check as a simple `if` in the setup phase.